components/schemes/README.md

```diff
@@ -10,119 +10,195 @@ path: /catalog/theming/
+containerScheme.typographyScheme = self.myTypographyScheme;
+```
+<!--</div>-->
 
-Consider where you will want to store your schemes so that they are accessible by your components.
-One solution is to have a global singleton that exposes shared scheme instances as we've done in
-the MDCCatalog, but the approach you take should be influenced by your app's architecture.
+### Using a subsystem scheme
 
 <!--<div class="material-code-render" markdown="1">-->
 #### Swift
 
 ```swift
-import MaterialColorScheme
-import MaterialShapeScheme
-import MaterialTypographyScheme
+import MaterialComponents.MaterialColorScheme
 
-let colorScheme = MDCSemanticColorScheme()
-let shapeScheme = MDCShapeScheme()
-let typographyScheme = MDCTypographyScheme()
+let colorScheme = MDCSemanticColorScheme(defaults: .material201804)
+// Configure custom properties to match your brand
+colorScheme.backgroundColor = .lightGray
 ```
 
 #### Objective-C
 
 ```objc
-#import "MaterialColorScheme.h"
-#import "MaterialTypographyScheme.h"
+#import <MaterialComponents/MaterialColorScheme.h>
 
 MDCSemanticColorScheme *colorScheme = [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-MDCShapeScheme *shapeScheme = [[MDCShapeScheme alloc] init];
-MDCTypographyScheme *typographyScheme = [[MDCTypographyScheme alloc] init];
+// Configure custom properties to match your brand
+colorScheme.backgroundColor = UIColor.lightGrayColor
 ```
 <!--</div>-->
 
-### Applying a scheme
+## Themers (Deprecated)
 
-To apply a scheme to a component you must first add the component's themers to your
-Podfile. You can see which themers a given component supports by looking at the component's src/
-directory.
+**Note** These will soon be deprecated for theming-extensions outlined above.
 
-```bash
-pod 'MaterialComponents/Buttons+ColorThemer'
-pod 'MaterialComponents/Buttons+ShapeThemer'
-pod 'MaterialComponents/Buttons+TypographyThemer'
-```
+Our approach to theming relies on the relationships between the following concepts:
+
+1. Components
+2. Schemes
+3. Themers
+
+Components are expected to provide public APIs for a variety of parameters. An example of a component is [MDCBottomNavigation](https://github.com/material-components/material-components-ios/tree/develop/components/BottomNavigation).
+
+Schemes represent a set of opinionated properties that are intended to be mapped to component parameters. There is a scheme for each Material Theming subsystem. For example, there is a scheme for the color, shape, and typography subsystems.
+
+Themers are objects that, when invoked with a scheme, will theme a component according to the [Material Design guidelines](https://material.io/design).
+
+## Examples
 
-You can now access the Button themers.
+### Theme component
 
 <!--<div class="material-code-render" markdown="1">-->
 #### Swift
 
 ```swift
+import MaterialComponents.MaterialBottomNavigation
 import MaterialComponents.MaterialButtons_ColorThemer
```

Comment:
Should this be ColorThemer or _Theming?

Reply:
It should be ColorThemer but the import should be `import MaterialComponents.MaterialBottomNavigation_ColorThemer`